examples/rust-cross-program-invocation/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+//! Rust example demonstrating invoking another program
+#![deny(missing_docs)]
+#![forbid(unsafe_code)]
+
+mod entrypoint;
+pub mod processor;
+
+solana_program::declare_id!("invoker111111111111111111111111111111111111");
```

Comment:
I feel like we should remove the `solana_program::declare_id!()` from all the examples because they setup the user for failure.  
* If they attempt to deploy the example to this address, it'll fail. 
* If they deploy the example to another address and accidentally use this address in their code, it'll fail. 
So this address only works in the context of a unit test

Reply:
Yeah, I originally had that, I'll go back,